client/src/components/Form/InputDropdown/InputDropdown.tsx

```diff
@@ -10,17 +10,31 @@ import { ReactComponent as ArrowDown } from '../formIcons/arrow-down.svg';
 import { ReactComponent as ArrowUp } from '../formIcons/arrow-up.svg';
 import './InputDropdown.scss';
 
-const InputDropdown = (props) => {
+
+interface InputDropdownProps {
+    dropdownTitle: string,
+    dropdownId: string,
+    inputValueSave:  (id: string, value: string) => void,
+    dropdownVisibility: (s: string) => void,
+    dropdownVisible: boolean,
+    inputFieldAs?: string,
+    dropdownArrow?: boolean,
+    toggleDropdown: () => void,
+    inputArray: Array<DropdownInputType>
+};
+
+const InputDropdown: React.FC<InputDropdownProps>  = (props) => {
```

Comment:
We should be more consistent. Like 1st deconstruct props, then state, then event handlers

Reply:
Aye aye Sir!